fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/digitalcash/DigitalCashActivity.java

```diff
@@ -39,25 +39,24 @@ protected void onCreate(Bundle savedInstanceState) {
     setupNavigationBar();
     setupBackButton();
 
+    subscribeToSelectItemEvents();
     setTheIntent();
```

Comment:
not your fault, but the `setTheIntent` function has terrible naming. We're not setting an intent, we're configuring the current view based on intent data. do you have any idea for a better name ?

Reply:
Just getIntentData would be already better and I'm sure there something more elegant. 
Nevertheless I hesitated before changing it and I decided not to because DigitalCash needs a complete "spring cleaning" with the most obvious being recurrent terrible naming (even java naming convention sometimes) and this is not the purpose of the current refactoring. 
So I would do it in a later PR if budget allows. If you disagree I will gladly do it on this one.

Reply:
Okay it triggered me as well so I changed just this one :)
Nevertheless the comment above subsist about digital cash in general